Integrations/Exabeam/Exabeam.py

```diff
@@ -0,0 +1,408 @@
+            # Get originating Exception in Exception chain
+            error_class = str(exception.__class__)
+            err_type = '<' + error_class[error_class.find('\'') + 1: error_class.rfind('\'')] + '>'
+            err_msg = '\nError Type: {}\nError Number: [{}]\nMessage: {}\n' \
+                      'Verify that the server URL parameter' \
+                      ' is correct and that you have access to the server from your host.' \
+                .format(err_type, exception.errno, exception.strerror)
+            raise DemistoException(err_msg, exception)
+
+    def _login(self):
+        """ Login using the credentials and store the cookie """
+        self._http_request('POST', '', full_url=self.server + '/api/auth/login', data={
+            'username': self.username,
+            'password': self.password
+        })
+
+    def _logout(self):
+        """ Logout from the session """
+        self._http_request('GET', self.server + '/api/auth/logout', None)
+
+    def test_module(self):
+        """
+        Performs basic get request to check if the server is reachable.
+        """
+        suffix_url = 'ping'
+        self._http_request('GET', suffix_url, resp_type='text')
+
+    def get_notable_users_request(self, api_unit=None, num=None, limit=None):
+
+        suffix_url = 'users/notable'
+
+        params = {
+            'unit': api_unit,
+            'num': num,
+            'numberOfResults': limit
+        }
+
+        response = self._http_request('GET', suffix_url, params)
+        return response
+
+    def get_user_info_request(self, username):
+
+        suffix_url = f'user/{username}/info'
+        response = self._http_request('GET', suffix_url)
+
+        return response
+
+    def get_watchlist_request(self):
+
+        suffix_url = 'watchlist'
+        response = self._http_request('GET', suffix_url)
+
+        return response
+
+    def get_peergroups_request(self):
+
+        suffix_url = 'peerGroup'
+
+        response = self._http_request('GET', suffix_url)
+        return response
+
+    def get_user_labels_request(self):
+
+        suffix_url = 'userLabel'
+        response = self._http_request('GET', suffix_url)
+
+        return response
+
+    def user_sequence_request(self, username=None, parse_start_time=None, parse_end_time=None):
+
+        suffix_url = f'user/{username}/sequences'
+        params = {
+            'username': username,
+            'startTime': parse_start_time,
+            'endTime': parse_end_time
+        }
+
+        response = self._http_request('GET', suffix_url, params)
+        return response
+
+
+''' COMMANDS + REQUESTS FUNCTIONS '''
+
+
+def get_notable_users(client: Client, args: Dict):
+    """ Get notable users in a period of time
+
+    Args:
+        client: Client
+        args: Dict
+
+    """
+    limit = args.get('limit')
+    time_period = args.get('time_period')
+    time_ = time_period.split(' ')
+    num = time_[0]
+    unit = time_[1]
```

Comment:
need to some verification here, like that `len(time_) == 2` and `api_unit in {'d', 'y', 'm', 'h'}`